### PR TITLE
dolphinEmuMaster: 5.0-14002 -> 5.0-14345

### PR DESCRIPTION
**Pull request by Kranzes**
###### Motivation for this change
Update to latest version at the time of making this PR.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ashkitten** [comment]
is there any reason you're updating to a non-beta version? it seems the last update was also not to a beta but in general we should stay on the beta updates because they are regular and reasonably well tested

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 125855` at e06f0e44 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/1d8cb571a389fd469a67b6852e9aa99a">dolphinEmuMaster</a></li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

File: pkgs/misc/emulators/dolphin-emu/master.nix [hunk 1]
```diff
@@ -21,13 +21,13 @@ let
...
-  version = "5.0-14002";
+  version = "5.0-14345";
 
   src = fetchFromGitHub {
     owner = "dolphin-emu";
     repo = "dolphin";
-    rev = "53222560650e4a99eceafcd537d4e04d1c50b3a6";
-    sha256 = "1m71gk9hm011fpv5hmpladf7abkylmawgr60d0czkr276pzg04ky";
+    rev = "2f8c9a77356b8eac8fdff0e9312fe2a8896c92b5";
+    sha256 = "0aw67drwsbswqcg18y1rx3di3hcq03pn7idk8s0mfi7sa8wja9i4";
```
Comment (SuperSandro2000) [inline pkgs/misc/emulators/dolphin-emu/master.nix:30]:
```suggestion
    sha256 = "0kg85jd58086argvljz8blhqfhfj716hp9jlr1vr593xp2vni2xl";
```

**Comment by Kranzes** [comment]
> is there any reason you're updating to a non-beta version? it seems the last update was also not to a beta but in general we should stay on the beta updates because they are regular and reasonably well tested

The name of the package is dolphinEmuMaster so i assumed that i should just use the latest commit from master, i also looked into the old commits in nixpkgs and they were also random latest (at the time) commits.

**Comment by ashkitten** [comment]
the main reason for keeping it on the beta releases is for netplay compatibility. this way, people using dolphin on nixos can (hopefully) be on the same revision as other package managers and be able to do netplay with them. i had intended to keep it updated with every beta, but that hasn't really worked out in practice.

**Comment by ashkitten** [comment]
> Maybe we should change the package name to dolphinEmuBeta instead of Master

i have no problem with this, if we keep the old name as an alias. i don't want to just break people's configs out of nowhere.

**Comment by Kranzes** [comment]
> > Maybe we should change the package name to dolphinEmuBeta instead of Master
> 
> i have no problem with this, if we keep the old name as an alias. i don't want to just break people's configs out of nowhere.

Sorry for the ignorance, but how would someone make an alias for something like that? also the PR title for both the update and the name change...

**Comment by ashkitten** [comment]
you can just do something like `dolphinEmuMaster = dolphinEmuBeta;`. the pr title isn't really that important, just split it into 2 commits one for the update and one for the name change.

**Comment by Kranzes** [comment]
> you can just do something like `dolphinEmuMaster = dolphinEmuBeta;`. the pr title isn't really that important, just split it into 2 commits one for the update and one for the name change.

where would one put that alias? could you give me an example?

**Comment by ashkitten** [comment]
you can put it in `pkgs/top-level/all-packages.nix` next to where `dolphinEmuBeta` is defined

**Comment by Kranzes** [comment]
> you can put it in `pkgs/top-level/all-packages.nix` next to where `dolphinEmuBeta` is defined

are you sure that you are not supposed to add it to `pkgs/top-level/aliases.nix`?

**Comment by ashkitten** [comment]
ah good point. i didn't remember about that file, oops

**Comment by Kranzes** [comment]
Closed due to personal laziness.

**Pull request closed without merging**